src/main/java/org/jfree/svg/SVGGraphics2D.java

```diff
@@ -2040,20 +1974,12 @@ private String registerClip(Shape clip) {
         return this.defsKeyPrefix + CLIP_KEY_PREFIX + index;
     }
     
-    private String transformDP(double d) {
-        if (this.transformFormat != null) {
-            return transformFormat.format(d);            
-        } else {
-            return String.valueOf(d);
-        }
+    private String transformDP(final double d) {
+        return doubleToString(d, ROUND_EVEN, transformDP);
     }
     
-    private String geomDP(double d) {
-        if (this.geometryFormat != null) {
-            return geometryFormat.format(d);            
-        } else {
-            return String.valueOf(d);
-        }
+    private String geomDP(final double d) {
+        return doubleToString(d, ROUND_EVEN, geometryDP);
```

Comment:
The profiler showed some time spent in this method. Removing the enumerated value would be a micro-optimization, as this is a critical part of the code.